README.md

```diff
@@ -1,46 +1,66 @@
-# Getting Started with Create React App and Redux
+## Monster Hunter World
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Available Scripts
+Web application to analyze the information about all the monsters of Monster Hunter World. You can search and see the data about each monster.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- Major languages: HTML, CSS, Javascript, React, Redux
+- Technologies used: GIT, Github, Gitflow, linters, Jest
+- [Monster Hunter World API](https://docs.mhw-db.com/)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo Link](https://radiant-taffy-25c53a.netlify.app/)
```

Comment:
- [ ] Kindly be noticed that the project is deployed properly, the home and detail page works as expected, and your link only displays hello.